docs/docs/widgets/radio-button.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 2
+---
+
+# Radio Button
+
+Radio Buttons can be used for to select one option in a 
+group of options.
+
+<img class="screenshot-full" src="/img/widgets/radio-button/widget.gif" alt="ToolJet - Radio Button Widget" height="420"/>
+
+:::tip
+It is preferred to use radio buttons when list of options
```

Comment:
`Radio buttons are preferred when the list of options is less than six and all the options can be displayed at once.`